skimage/segmentation/tests/test_active_contour_model.py

```diff
@@ -3,9 +3,12 @@
 from skimage.color import rgb2gray
 from skimage.filters import gaussian
 from skimage.segmentation import active_contour
-from numpy.testing import assert_equal, assert_allclose, assert_raises
+from skimage.segmentation.active_contour_model import new_scipy
```

Comment:
Actually, nevermind. I thought this is used somewhere else, but, apparently, is not.